integrators/ordinary_differential_equations.hpp

```diff
@@ -37,9 +42,13 @@ struct SpecialSecondOrderDifferentialEquation {
     std::vector<Displacement> position_error;
     std::vector<Velocity> velocity_error;
   };
+  // A functor that computes f(q, t) and stores it in |*accelerations|.
+  // |*accelerations| must have the same size as |positions|.
   RightHandSideComputation compute_acceleration;
 };
 
+// An initial value problem, together with a final time for the solution
```

Comment:
We'll, it's the [standard terminology](http://en.wikipedia.org/wiki/Initial_value_problem), I'm not going to put a Calculus I course in there...